indico/modules/events/registration/views.py

```diff
@@ -24,7 +24,18 @@
 
 class WPManageRegistration(WPJinjaMixin, WPConferenceModifBase):
     template_prefix = 'events/registration/'
-    sidemenu_option = 'registration'
+
+    @property
+    def sidemenu_option(self):
+        if self._conf.getType() != 'conference':
```

Comment:
Because I would need to `return 'registration'` twice in that case. Do you prefer this?

```python
@property
def sidemenu_option(self):
    if self._conf.getType() == 'conference':
        return 'registration'
    regform = self._kwargs.get('regform')
    if not regform:
        registration = self._kwargs.get('registration')
        if registration:
            regform = registration.registration_form
    if regform and regform.is_participation:
        return 'participants'
    return 'registration'
```